packages/yoroi-extension/app/components/wallet/send/WalletSendFromSteps/WalletSendPreviewStep.js

```diff
@@ -0,0 +1,387 @@
+      .shiftedBy(-tokenInfo.Metadata.numberOfDecimals);
+
+    const coinPrice = this.props.getCurrentPrice(
+      tokenInfo.Identifier,
+      toCurrency
+    );
+
+    if (coinPrice == null) return '-';
+
+    return calculateAndFormatValue(
+      shiftedAmount,
+      coinPrice
+    );
+  }
+
+  renderSingleAmount: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.amount}>
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
+            <span className={styles.currencySymbol}>
+              &nbsp;{unitOfAccountSetting.currency}
+            </span>
+          </div>
+          <div className={styles.amountSmall}>{formatValue(entry)}
+            <span className={styles.currencySymbol}>&nbsp;{
+              truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+            }
+            </span>
+          </div>
+        </>
+      ) : (
+        <div className={styles.amount}>{formatValue(entry)}
+          <span className={styles.currencySymbol}>&nbsp;{
+            truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+          }
+          </span>
+        </div>
+      );
+  }
+  renderTotalAmount: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.totalAmount}>
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
+            <span className={styles.currencySymbol}>
+              &nbsp;{unitOfAccountSetting.currency}
+            </span>
+          </div>
+          <div className={styles.totalAmountSmall}>{formatValue(entry)}
+            <span className={styles.currencySymbol}>&nbsp;{
+              truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+            }
+            </span>
+          </div>
+        </>
+      ) : (
+        <div className={styles.totalAmount}>{formatValue(entry)}
+          <span className={styles.currencySymbol}>&nbsp;{
+            truncateToken(getTokenName(this.props.getTokenInfo(entry)))
+          }
+          </span>
+        </div>
+      );
+  }
+  renderSingleFee: TokenEntry => Node = (entry) => {
+    const formatValue = genFormatTokenAmount(this.props.getTokenInfo);
+
+    const { unitOfAccountSetting } = this.props;
+    return unitOfAccountSetting.enabled
+      ? (
+        <>
+          <div className={styles.fees}>+
+            {this.convertedToUnitOfAccount(entry, unitOfAccountSetting.currency)}
+            <span className={styles.currencySymbol}>
+              &nbsp;{unitOfAccountSetting.currency}
+            </span>
+          </div>
+          <div className={styles.feesSmall}>
```

Comment:
Is this meant to be smaller? Semes it's the equal font.

Also, are you sure to put fiat amount before ADA amount?

Reply:
Should look like this! 
<img width="700" alt="Screen Shot 2022-01-30 at 3 51 24 AM" src="https://user-images.githubusercontent.com/72753578/151683634-a408ce17-3a70-40bd-9e98-d019a2a2fd24.png">

and this 
<img width="486" alt="Screen Shot 2022-01-30 at 3 51 56 AM" src="https://user-images.githubusercontent.com/72753578/151683651-221dd718-a133-430e-bdec-07ecfccf4bcd.png">

Reply:
Currently I looks like this:
![confirmation](https://user-images.githubusercontent.com/1028789/152364175-021e1513-a426-4227-a12e-0e4faca0e5c3.jpg)
So please fix.

And there is no definition of `feesSmall` in `WalletSendPreviewStep.scss`.